### PR TITLE
Timeline Module for Beaver Builder Enhancement

### PR DESCRIPTION
**Pull request by bhushanb20**
* PHPCS errors and warnings fixed
-------------------------------------------------------------------------------------------
* Old: Timeline for Beaver Builder > New: Timeline Module for Beaver Builder
-------------------------------------------------------------------------------------------
Version 1.0.1 - 
* Added latest option for Date / Content Area for more timeline design possibility.
* Added new customization options for timeline section design.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,30 +1,94 @@
...
+=== Timeline Module for Beaver Builder ===
 Contributors: brainstormforce, bhushansbobade20
 Donate link: https://www.brainstormforce.com/payment/
 Requires at least: 4.4
-Tags: beaver builder, page builder timeline, bb timeline, timelines, drag and drop timeline, Responsive timeline, Responsive, timeline, Animation, Animated Timeline.
-Stable tag: 1.0.0
-Tested up to: 4.7
+Tags: beaver builder, page builder timeline, bb timeline, timelines, drag and drop timeline, Timeline Module for Beaver Builder
+Stable tag: 1.0.1
+Tested up to: 4.6
```
Comment (Nikschavan) [inline readme.txt]: @bhushanb20 update the latest WordPress version for tested up to.

File: readme.txt [hunk 2]
```diff
@@ -1,30 +1,94 @@
...
 
 = 1.0.0 =
-- Initial release
\ No newline at end of file
+- Initial release
+
+= 1.0.1 =
+* We have added new option for Date / Content Area for more timeline design possibility.
+* Also have added new customization option for timeline section design.
+
```
Comment (Nikschavan) [inline readme.txt]: The newer versions should be above in the changelog.

File: readme.txt [hunk 3]
```diff
@@ -1,30 +1,94 @@
...
+
+== Upgrade Notice ==
+
+= 1.0.1 =
+This version added new options. Upgrade immediately for more timeline design possibility.
```
Comment (Nikschavan) [inline readme.txt]: No need to add an upgrade notice.

File: timeline-for-beaver-builder-module/css/frontend.css [hunk 4]
```diff
@@ -17,6 +17,15 @@
 /*main heading align /.*/
 
 /*-----------------------------Common-CSS-Start---------------------------------------*/
+
+#fl-builder-settings-section-timeline_date_area {
+    margin-bottom: 0 !important;
```
Comment (Nikschavan) [inline timeline-for-beaver-builder-module/css/frontend.css]: Is it possible to not use `!important`?

File: timeline-for-beaver-builder-module/css/frontend.css [hunk 5]
```diff
@@ -256,7 +262,7 @@
     position: absolute;
     pointer-events: none;
     border-width: 10px;
-    top: 10px;
+    /*top: 10px;*/
```
Comment (Nikschavan) [inline timeline-for-beaver-builder-module/css/frontend.css]: if this CSS is not required, delete it, don't commit commented code.

File: timeline-for-beaver-builder-module/timeline-for-beaver-builder-module.php [hunk 6]
```diff
@@ -744,9 +745,34 @@ public function update($settings)
...
+                                )
+                            ),
+                        ),
+                    ),
+                ),
+
                 //Set Date
                 'date'       => array(
-                    'title'         => __( 'Date', 'bb-timeline' ),
+                    //'title'         => __( 'Date', 'bb-timeline' ),
```
Comment (Nikschavan) [inline timeline-for-beaver-builder-module/timeline-for-beaver-builder-module.php]: Don't commit commented code.

**Review by Nikschavan — CHANGES_REQUESTED** [review]

**Comment by Nikschavan** [comment]
@bhushanb20 The PHPCS requirements are also required to be fixed before we merge this.

File: readme.txt (hunk 1 above)
Comment (bhushanb20) [reply readme.txt]: I have updated the WordPress version tested up 4.7

File: readme.txt (hunk 2 above)
Comment (bhushanb20) [reply readme.txt]: I have listed versions from most recent at top to oldest at bottom.

File: readme.txt (hunk 3 above)
Comment (bhushanb20) [reply readme.txt]: I have removed update notice

File: timeline-for-beaver-builder-module/css/frontend.css (hunk 5 above)
Comment (bhushanb20) [reply timeline-for-beaver-builder-module/css/frontend.css]: Done

File: timeline-for-beaver-builder-module/css/frontend.css (hunk 4 above)
Comment (bhushanb20) [reply timeline-for-beaver-builder-module/css/frontend.css]: I have removed !important from css

File: timeline-for-beaver-builder-module/timeline-for-beaver-builder-module.php (hunk 6 above)
Comment (bhushanb20) [reply timeline-for-beaver-builder-module/timeline-for-beaver-builder-module.php]: Removed commented code

**Review by Nikschavan — APPROVED** [review]

**Pull request merged**